pyproject.toml

```diff
@@ -4,16 +4,21 @@ version = "3.0.3"
 description = "Python wrapper and metaschema for datadictionary."
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/uc-cdis/dictionaryutils" 
 
 [tool.poetry.dependencies]
 python = "^3.6"
+PyYAML = "~=5.1"
+jsonschema = ">=2.5,<3.0"
+cdislogging = "^1.0.0"
+requests = "~=2.18"
 
 [tool.poetry.dev-dependencies]
-nose = ">=1.3.7<2.0.0"
+nose = ">=1.3.7,<2.0.0"
 rstr = "2.2.6"
-cdisutils = {git = "https://github.com/NCI-GDC/cdisutils.git", rev = "1.4.0"}
 cdislogging = "^1.0.0"
```

Comment:
i think `cdislogging` should be in `dependencies` - it's used outside of the tests